types/leaflet-side-by-side/index.d.ts

```diff
@@ -0,0 +1,35 @@
+// Type definitions for react-side-effect 2.2
+// Project: https://github.com/digidem/leaflet-side-by-side
+// Definitions by: Remo H. Jansen <https://github.com/OliverBiel/>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
```

Comment:
Forgot to change the name of the package (Leaflet Side by Side) and my name (Gabriel O. Martins)

Reply:
so can you make this change? Thanks!